yarn.lock

```diff
@@ -33,20 +33,20 @@
 
 "@babel/code-frame@^7.23.5", "@babel/code-frame@^7.24.1", "@babel/code-frame@^7.24.2":
   version "7.24.2"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.24.2.tgz#718b4b19841809a58b29b68cde80bc5e1aa6d9ae"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.24.2.tgz"
```

Comment:
ooh wait can we undo this yarn => npm registry change?

Reply:
Weird! Good catch! I accidentally ran `npm i` at some point, but subsequent `yarn` runs must not undo that. It's fixed now!